.github/workflows/README.md

```diff
@@ -10,7 +10,7 @@ Each app should have:
 
 - `ci-[app_name]`: must be created; should run linting and testing
 - `ci-[app_name]-vulnerability-scans`: calls `vulnerability-scans`
-  - Based on [ci-app-vulnerability-scans](https://github.com/navapbc/template-infra/blob/main/.github/workflows/ci-app-vulnerability-scans.yml)
+  - Based on [ci-app-vulnerability-scans.yml.jinja](./ci-{{app_name}}-vulnerability-scans.yml.jinja)
```

Comment:
I don't think these links will render correctly because `{{app_name}}` -- click on the link at https://github.com/navapbc/labs-decision-support-tool/blob/DST-709-increase-dev-memory/.github/workflows/README.md